packages/mui-material/src/index.d.ts

```diff
@@ -474,3 +474,12 @@ export * from './GlobalStyles';
  * @deprecated will be removed in v5.beta, please use StyledEngineProvider from @mui/material/styles instead
  */
 export { StyledEngineProvider } from './styles';
+
+export { default as unstable_composeClasses } from '@mui/base/composeClasses';
+export * from '@mui/base/composeClasses';
```

Comment:
Do we need this line?

Reply:
You're right. I copied it from @mui/base exports but it looks like it's not needed (same as below).